src/components/LoadingMessage.css

```diff
@@ -1,15 +1,31 @@
 .loading-message {
-  height: 50px;
-  width: 100%;
-  border: 1px solid black;
   z-index: 1;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  display: inline;
+  /* justify-content: center;
+  align-items: center; */
 }
 
-h1 {
-  font: 10px;
-  padding: 0px;
+.loading-icon {
   margin: 0px;
+  padding: 0px;
+  display: inline-flex;
+  align-self: center;
+  width: 15px;
+  height: 15px;
+  border: 3px solid black;
+  border-radius: 50%;
+  border-top-color: pink;
+  animation: spin 1s ease-in-out infinite;
+  -webkit-animation: spin 1s ease-in-out infinite;
+}
+
+@keyframes spin {
+  to {
+    -webkit-transform: rotate(360deg);
+  }
+}
+@-webkit-keyframes spin {
```

Comment:
I wonder how old of a device you need to have to need this fallback? Could probably get away with removing it? 🤔